locale/bn.yml

```diff
@@ -0,0 +1,1813 @@
+eval: ""
+exit: ""
+extends: ""
+final: ""
+for: ""
+foreach: ""
+function: ""
+global: ""
+goto: ""
+if: ""
+implements: ""
+include: ""
+include_once: ""
+instanceof: ""
+insteadof: ""
+interface: ""
+isset: ""
+list: ""
+namespace: ""
+new: ""
+or: ""
+print: ""
+private: ""
+protected: ""
+public: ""
+require: ""
+require_once: ""
+return: ""
+static: ""
+switch: ""
+throw: ""
+trait: ""
+try: ""
+unset: ""
+use: ""
+var: ""
+while: ""
+xor: ""
+__CLASS__: ""
+__DIR__: ""
+__FILE__: ""
+__FUNCTION__: ""
+__LINE__: ""
+__METHOD__: ""
+__NAMESPACE__: ""
+__TRAIT__: ""
+
+# << Error Messages >>
+
+# ----------------------
+# |  LANGUAGE: PYTHON  |
+# ----------------------
+# << Keywords >>
+  False: "মিথ্যা"
+  None: "কোনোটিই নয়"
+  True: "সত্য"
+  and: "এবং"
+  as: "হিসাবে"
+  assert: "দাবি করা"
+  async: "অসমনিয়ত"
+  await: "অপেক্ষা করা"
+  break: "বিরতি"
+  class: "বিভাগ"
+  continue: "বজায় রাখা"
+  def: "নির্ধারণ"
+  del: "মুছে ফেলা"
+  elif: "অন্যথায় যদি"
+  else: "অন্যথায়"
+  except: "বাদে"
+  finally: "অবশেষে"
+  for: "জন্য"
+  from: "হইতে"
+  global:  "বিশ্বব্যাপী"
+  if: "যদি"
+  import: "আমদানি"
+  in: "মধ্যে"
+  is: "হয়"
+  lambda: "বেনামী"
+  nonlocal: "অস্থানীয়"
+  not: "নয়"
+  or: "অথবা"
+  pass: "ছাড়পত্র"
+  raise: "সৃষ্টি করা"
+  return: "প্রত্যাবর্তন"
+  try: "চেষ্টা"
+  while: "যখন"
+  with: "সঙ্গে"
+  yield: "ফলন করা"
+
+# << Built-In Functions >>
+
+  abs: "নিরপেক্ষ মূল্য"
+  all: "সমস্ত"
+  any: "যেকোনো"
+  ascii: "অ্যাসকি"
+  bin: "বাইনারি উপস্থাপনা"
+  bool: "বুলিয়ান"
+  breakpoint: "ব্রেকপয়েন্ট"
+  bytearray: "প্রদত্ত বাইটের অ্যারে"
+  bytes: "বাইট"
+  callable: "কলযোগ্য"
+  chr: "প্রতিনিধিত্বকারী অক্ষর"
+  classmethod: "ক্লাস পদ্ধতি"
+  compile: "কম্পাইল"
+  complex: "জটিল"
+  delattr: "বৈশিষ্ট্য মুছে ফেলুন"
+  dict: "অভিধান"
+  dir: "সমস্ত বৈশিষ্ট্য এবং পদ্ধতি"
+  divmod: "ভাগফল এবং তাদের বিভাগের অবশিষ্টাংশে"
+  enumerate: "গোনা"
+  eval: "মূল্যায়ন"
+  exec: "করণ"
+  filter: "ফিল্টার"
+  float: "ফ্লোট"
+  format: "পদ্ধতি"
+  frozenset: "তাদের অপরিবর্তনীয় করে তোলে"
+  getattr: "মান ফেরত দিন"
+  globals: "বিশ্বব্যাপী"
+  hasattr: "যদি বস্তুর বৈশিষ্ট্য থাকে"
+  hash: "হ্যাশ"
+  help: "সাহায্য"
+  hex: "হেক্স"
+  id: "আইডি"
+  input: "ইনপুট"
+  int: "পূর্ণসংখ্যা"
+  isinstance: "যদি নির্দিষ্ট প্রকার"
+  issubclass: "যদি সাবক্লাস"
+  iter: "পুনরাবৃত্তি"
+  len: "দৈর্ঘ্য"
+  list: "তালিকা"
+  locals: "স্থানীয়দের"
+  map: "মানচিত্র"
+  max: "সর্বোচ্চ"
+  memoryview: "মেমরিভিউ"
+  min: "ন্যূনতম"
+  next: "পরবর্তী"
+  object: "বস্তু"
+  oct: "অক্টাল মান"
+  open: "খোলা"
+  ord: "ইউনিকোড কোড"
+  pow: "গুণ"
+  print: "ছাপ"
+  property: "বৈশিষ্ট্য"
+  range: "পরিসীমা"
+  repr: "স্ট্রিং প্রতিনিধিত্ব"
+  reversed: "বিপরীত"
+  round: "সুসম্পন্ন করা"
+  set: "সেট"
+  setattr: "বৈশিষ্ট্য সেট করে"
+  slice: "টুকরা"
+  sorted: "সাজানো"
+  staticmethod: "স্ট্যাটিক পদ্ধতি"
+  str: "স্ট্রিং"
+  sum: "যোগফল"
+  super: "সুপার"
+  tuple: "টুপল"
+  type: "প্রকার"
+  vars: "যার"
+  zip: "জিপ"
+  __import__: "__আমদানি__"
```

Comment:
This is fantastic @SandipPalit ! Thank you so much for adding the initial translation 👍  

As software always iterates, we found a better way to mark translation reviews from Hacktoberfest feedback. I'm still working on documenting the new process, but would you mind making the following change?

- [ ] Add your @ username in a new line under each translated word.
- [ ] Provide the emoji that best captures your confidence around the translation (see key in [Translation Guide: Step 7](https://docs.legesher.io/legesher-translations/translation-guide))
- [ ] Add any additional commentary or other words you considered in your own review line

Here's an example:
``` yml
  zip: "জিপ"
    # ✅  @SandipPalit
  __import__: "__আমদানি__"
    # ❓  @SandipPalit what about "this" ? or "that"?
```

Let me know if you have any questions!

Reply:
Hi @madiedgar, I have implemented the following changes.

- [x] Add your @ username in a new line under each translated word.
- [x] Provide the emoji that best captures your confidence around the translation.
- [x] Add any additional commentary or other words you considered in your own review line.

Please let me know if I can do even more improvements.